mandatory/3-function-output.js

```diff
@@ -1,14 +1,21 @@
-// Add comments to explain what this function does. You're meant to use Google!
+/*Add comments to explain what this function does. You're meant to use Google!
+
+ This function it's going to take a random number from 0 to 1(excluding ) and it's going to multiply that
+number by 10 */
 function getRandomNumber() {
   return Math.random() * 10;
-}
+}  
 
-// Add comments to explain what this function does. You're meant to use Google!
+/*Add comments to explain what this function does. You're meant to use Google!
+
+The concat method joins 2 or more strings together,without  changing the existing string but 
+ by returning a new string */
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
+    return firstWord.concat(" ",secondWord.concat(" ",thirdWord));
```

Comment:
The function `concat` can accept multiple parameters. Can you think of a way on how to improve this?